xyz.anirban.ChatGPTElectron.yml

```diff
@@ -0,0 +1,63 @@
+app-id: xyz.anirban.ChatGPTElectron
+runtime: org.freedesktop.Platform
+runtime-version: '22.08'
+sdk: org.freedesktop.Sdk
+base: org.electronjs.Electron2.BaseApp
+base-version: "22.08"
+command: run.sh
+rename-desktop-file: chatgpt-electron.desktop
+rename-icon: chatgpt-electron
+finish-args:
+  - --share=ipc
+  - --share=network
+  - --socket=x11
+  - --socket=pulseaudio
+  - --device=dri
+  # - --filesystem=home
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.node18
+build-options:
+  append-path: /usr/lib/sdk/node18/bin
+modules:
+
+  - name: run
+    buildsystem: simple
+    build-commands:
+      - install -D run.sh /app/bin/run.sh
+    sources:
+      - type: script
+        dest-filename:  run.sh
+        commands:
+          - 'echo "Running Inside Sandbox" '
+          - 'export TMPDIR="$XDG_RUNTIME_DIR/app/${FLATPAK_ID}"'
+          - 'exec zypak-wrapper /app/opt/chatgpt-electron/chatgpt-electron-linux-x64/chatgpt-electron'
+
+  - name: assets
+    buildsystem: simple
+    build-commands:
+      - cp -r chatgpt-electron/usr/share /app
+      - desktop-file-edit --set-key=Exec --set-value="run.sh" /app/share/applications/chatgpt-electron.desktop
+      - desktop-file-edit --set-key=Icon --set-value="xyz.anirban.ChatGPTElectron" /app/share/applications/chatgpt-electron.desktop
+
+    sources:
+      - type: git
+        url: "https://github.com/anirbandey1/project-assets.git"
+        branch: main
+
+  
+  - name: chatgpt-electron
+    buildsystem: simple
+    build-commands:
+      - tar xzvf chatgpt-electron.tar.gz
+      - rm chatgpt-electron.tar.gz
+      - mkdir -pv /app/opt/chatgpt-electron
+      - cp -r * /app/opt/chatgpt-electron
+
+    sources:
+      - type: file
+        url: "https://github.com/anirbandey1/chatgpt-electron/releases/download/v1.0.0/chatgpt-electron-linux-x64.tar.gz"
```

Comment:
so how is this gonna work on aarch64?

Reply:
it won't work on aarch64 as of now 
i think i should just specify it to be

Reply:
Made it x86 only